playbooks/Installation/disconnected_environments.adoc

```diff
@@ -11,28 +11,420 @@ There are at least three considerations for enterprise environments:
 
 == Disconnected Environments
 
-=== Yum Repositories
+=== Assumptions
 
-If the environment does not use Red Hat Satellite 6, you will need to set up a server to host yum repositories locally. That one server alone needs to be registered with Red Hat Subscription Manager.
+This document assumes that you understand the overall architecture of OpenShift 3 and that you have already planned out what the topology of your environment will look like. If you do not have a thorough understanding of the architecture of OpenShift 3, the service layer, projects, and how topologies are configured via the scheduler, there are steps in this document that may not make sense. This document also assumes that you have some understanding of Docker, registries, and Docker’s command line tools.
```

Comment:
We should work in a link here to the OSE Architecture Overview: https://docs.openshift.com/enterprise/3.0/architecture/overview.html